Sources/Web3Auth/String+extensions.swift

```diff
@@ -30,3 +29,60 @@ extension String {
         return result
     }
 }
+
+extension String {
+    func fromBase64() -> String? {
+        guard let data = Data(base64Encoded: self) else {
+            return nil
+        }
+        return String(data: data, encoding: .utf8)
+    }
+
+    func toBase64() -> String {
+        return Data(utf8).base64EncodedString()
+    }
+
+    func strip04Prefix() -> String {
+        if hasPrefix("04") {
+            let indexStart = index(startIndex, offsetBy: 2)
+            return String(self[indexStart...])
+        }
+        return self
+    }
+
+    func strip0xPrefix() -> String {
+        if hasPrefix("0x") {
+            let indexStart = index(startIndex, offsetBy: 2)
+            return String(self[indexStart...])
+        }
+        return self
+    }
+
+    func addLeading0sForLength64() -> String {
```

Comment:
call this function padStart and take character, total length as input.

Refer [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) for fn definition

Reply:
done